XmlSchemaValidationTest fails on the different language settings

I'm running the tests on my local machine which is using `zh_CN.utf-8`. Then `XmlSchemaValidationTest` is failing with

```
[ERROR]   XmlSchemaValidationTest.addCheckResultInvalid:74 
Expecting throwable message:
  "Unmarshalling Error: cvc-maxInclusive-valid: 对于类型为 'smallIntType' 的 maxInclusive '1024', 值 '1025' 不具有面有效性。 "
to contain:
  "arshalling Error: cvc-maxInclusive-valid: Value '1025' is not facet-valid with respect to maxInclusive '1024' for type 'smallIntType'."
but did not.
```

The cause message looks like from `i18n` resource bunlde and it transforms some words to Chinese. This can be fixed by setting `LANG` environment.

--- a/docs/modules/ROOT/examples/client-server/application.properties
+++ b/docs/modules/ROOT/examples/client-server/application.properties
@@ -81,3 +81,5 @@ quarkus.cxf.client.basicAuthSecureWsdl.username = bob
 quarkus.cxf.client.basicAuthSecureWsdl.password = ${client-server.bob.password}
 quarkus.cxf.client.basicAuthSecureWsdl.secure-wsdl-access = true
 # end::client-basic-auth-wsdl[]
+
+quarkus.default-locale = en_US

--- a/integration-tests/client-server/src/main/resources/application.properties
+++ b/integration-tests/client-server/src/main/resources/application.properties
@@ -81,3 +81,5 @@ quarkus.cxf.client.basicAuthSecureWsdl.username = bob
 quarkus.cxf.client.basicAuthSecureWsdl.password = ${client-server.bob.password}
 quarkus.cxf.client.basicAuthSecureWsdl.secure-wsdl-access = true
 # end::client-basic-auth-wsdl[]
+
+quarkus.default-locale = en_US

--- a/pom.xml
+++ b/pom.xml
@@ -199,13 +199,28 @@
                     <configuration>
                         <systemPropertyVariables>
                             <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
+                            <user.language>en</user.language>
+                            <user.country>US</user.country>
                         </systemPropertyVariables>
+                        <environmentVariables>
+                            <LANG>en_US.UTF-8</LANG>
+                        </environmentVariables>
                     </configuration>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-failsafe-plugin</artifactId>
                     <version>${maven-surefire-plugin.version}</version>
+                    <configuration>
+                        <systemPropertyVariables>
+                            <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
+                            <user.language>en</user.language>
+                            <user.country>US</user.country>
+                        </systemPropertyVariables>
+                        <environmentVariables>
+                            <LANG>en_US.UTF-8</LANG>
+                        </environmentVariables>
+                    </configuration>
                 </plugin>
                 <plugin>
                     <groupId>org.l2x6.rpkgtests</groupId>

